mono/mini/mini.c

```diff
@@ -4125,6 +4125,11 @@ mono_jit_compile_method_inner (MonoMethod *method, MonoDomain *target_domain, in
 
 	mono_domain_lock (target_domain);
 
+	if (mono_stats_method_desc && mono_method_desc_full_match (mono_stats_method_desc, method)) {
+		g_printf ("Printing JIT stats at method: %s\n", mono_method_get_full_name (method));
```

Comment:
nit: It prints a slightly different string than in the interpreter case (which is more general, and more "correct").

Maybe we can move the if check and `g_printf` into `mono_runtime_print_stats`, so we could get rid of `extern mono_stats_method_desc`.